resume.md

```diff
@@ -1,49 +1,46 @@
-* ...
-
-## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
-
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+Objective
+--------------------------------------------------------
+To obtain an entry-level position in software engineering in order to obtain industry experience, expand my knowledge, and maximize my skillset.
+
+Education
+--------------------------------------------------------
+University of Southern California, Los Angeles CA	Aug 2017 – June 2021 (Expected)
+Bachelor of Science, Mechanical Engineering
+Minor in Computer Programming
+GPA: 3.47, Dean’s List: 2 semesters	
+Computer Science Coursework: Programming Systems Design  •  Analysis of Algorithms
+
+Programming Skills
+--------------------------------------------------------
+Languages: C++  •  Java •  HTML  •  CSS  •  PHP  •  JavaScript  •  SQL  •  MATLAB
+Technologies: React •  Bootstrap  •  MongoDB  •  AWS  •  Git  •  Siemens NX  
+
+Experience
+--------------------------------------------------------
+Teaching Assistant	Aug 2019 – present
+USC Viterbi School of Engineering	Los Angeles, CA
+•	Assisted two professors for Object Oriented Programming and Professional C++
+•	Administered weekly office hours which improved debugging and communication skills
+
+Enterprise Applications Intern	June 2019 – Oct 2019
+Los Angeles Unified School District	Los Angeles, CA
+•	Consulted the Information Technology Desk Department and managed their pages using HTML, CSS, and JS to ensure responsive design and ADA compliancy
+•	Established the LAUSD hall of fame of 2,000+ alumni by implementing a web form using HTML, CSS, and PHP to input data into a SQL Server database
+
+Academic Projects	
+--------------------------------------------------------
+Sorting Algorithm Visualizer/Timer	July 2020 – July 2020
+•	Created an app to visualize/time merge, heap, quick, bubble, selection, and insertion sort
```

Comment:
why kind of app using what technology?  is this native?  is this web?  

sounds like a school project - folks from your school will also have this on their resume, consider swapping for a more personalized project.